modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemIcon from '@elementor/ui/ListItemIcon';
+import ListItemText from '@elementor/ui/ListItemText';
+import Collapse from '@elementor/ui/Collapse';
+import ChevronDownIcon from '@elementor/icons/ChevronDownIcon';
+import RadioButtonUncheckedIcon from '@elementor/icons/RadioButtonUncheckedIcon';
+import CardMedia from '@elementor/ui/CardMedia';
+import Link from '@elementor/ui/Link';
```

Comment:
```suggestion
import { Button, Card, CardContent, CardActions, Typography, ListItemButton, ListItemIcon, ListItemText, Collapse, CardMedia, Link } from '@elementor/ui';
import { ChevronDownIcon, RadioButtonUncheckedIcon } from '@elementor/icons';
```
Maybe? I guess @rotemee would have already flagged it if it was an issue.

Reply:
@louiswol94 
https://mui.com/material-ui/guides/minimizing-bundle-size/

Reply:
so I remember Rotem said it is better to import 1 by 1.